pset-iterators.js

```diff
@@ -93,6 +110,18 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+let presidentialParties = []
+
+const findpresidentialParties = presidents.filter(function (president) {
+
+  presidentialParties.push(president.party)
+
+})
+
+console.log(presidentialParties)
+console.log(presidentialParties.length)
```

Comment:
@thetripler Good work on this solution here. You were able to get back all of the presidential parties and display them.  One suggestion when you're using the method `filter` this will return an array of objects based on the criteria you provided. In this case, you didn't need to set a variable `presedentialParties` and  ` presidentialParties.push(president.party)`, you could've replaced ` presidentialParties.push(president.party)` with the following `return president.party`. Also, the `filter` method returns the entire object, there is a method `map` that would've returned each party for you in an array. Check out some info on the `map` method [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)

Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L94)